.github/workflows/comment_automation.yml

```diff
@@ -0,0 +1,60 @@
+name: Automated Issue Responses
+
+on:
+  issues:
+    types: [opened, labeled]
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Automated Comment on Issue Creation
+        if: github.event.action == 'opened'
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const issueComment = `Hello @${{ github.actor }}, thanks for the issue report! Your report is being reviewed and will be addressed as soon as possible.`;
```

Comment:
`Hello @${{ github.actor }}, thanks for the issue report! We are reviewing your report and will address it as soon as possible.